usecases/schema/handler_test.go

```diff
@@ -42,7 +44,6 @@ var schemaTests = []struct {
 func testAddObjectClass(t *testing.T, handler *Handler, fakeMetaHandler *fakeMetaHandler) {
 	t.Parallel()
 
-	fakeMetaHandler.On("ReadOnlySchema").Return(models.Schema{})
```

Comment:
non-blocking: what is the reasong we're removing those. They were unnecessary because we were not calling the assert called ?

Reply:
exactly yes